optaplanner-quickstarts/quarkus-school-timetabling/build.gradle

```diff
@@ -14,9 +14,10 @@ repositories {
 }
 
 dependencies {
-    implementation enforcedPlatform("io.quarkus:quarkus-universe-bom:1.4.2.Final")
-    implementation 'io.quarkus:quarkus-optaplanner'
-    implementation 'io.quarkus:quarkus-optaplanner-jackson'
+    implementation platform("io.quarkus:quarkus-universe-bom:1.7.0.Final")
```

Comment:
What's the difference between "enforcedPlatform" and "platform"?

Reply:
"platform" works pretty much the same way as importing a BOM in maven. "enforcedPlatform" overrides whatever is declared before or after that => the BOM always wins.